compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ConstantValueResolver.java

```diff
@@ -122,6 +122,12 @@ public void visit(BLangSimpleVarRef varRef) {
             return;
         }
 
+        if (!this.unresolvedConstants.containsKey(varRef.symbol)) {
+            dlog.error(varRef.pos, DiagnosticErrorCode.CANNOT_RESOLVE_CONST, varRef.symbol.name.value);
```

Comment:
Shall we use `constSymbol` instead of `varRef.symbol`.

Let's update L132 also.